docs/self-managed/platform-deployment/helm-kubernetes/platforms/amazon-eks/irsa.md

```diff
@@ -0,0 +1,390 @@
+- Via the REST API.
+
+The important part is assigning the `iam_role_arn` of the previously created `opensearch_role` to an internal role within OpenSearch. For example, `all_access` on the OpenSearch side is a good candidate, or if required, extra roles can be created with more restrictive access.
+
+### Operate
+
+Configure Operate to use the feature set of IRSA for the OpenSearch Exporter. Check the [Operate OpenSearch configuration](../../../../operate-deployment/operate-configuration.md#elasticsearch-or-opensearch).
+
+#### Kubernetes Configuration
+
+As an example, configure the following environment variables:
+
+```
+- name: CAMUNDA_OPERATE_ELASTICSEARCH_URL
+  value: https://test-domain.region.es.amazonaws.com
+- name: CAMUNDA_OPERATE_ZEEBEELASTICSEARCH_URL
+  value: https://test-domain.region.es.amazonaws.com
+```
+
+Where the value is whatever the endpoint of your OpenSearch cluster is.
+
+:::note
+AWS OpenSearch listens on port 443 opposed to the usual port 9200.
+:::
+
+:::note
+Don't forget to set the `serviceAccountName` of the deployment/statefulset to the created service account with the IRSA annotation.
+:::
+
+### Zeebe
+
+Configure Zeebe to use the feature set of IRSA for the OpenSearch Exporter. Check the [Zeebe OpenSearch exporter configuration](../../../../zeebe-deployment/configuration/broker.md#zeebebrokerexportersopensearch-opensearch-exporter).
+
+#### Kubernetes Configuration
+
+As an example, configure the following environment variables:
+
+```yaml
+- name: ZEEBE_BROKER_EXPORTERS_OPENSEARCH_ARGS_AWS_ENABLED
+  value: true
+- name: ZEEBE_BROKER_EXPORTERS_OPENSEARCH_CLASSNAME
+  value: io.camunda.zeebe.exporter.opensearch.OpensearchExporter
+- name: ZEEBE_BROKER_EXPORTERS_OPENSEARCH_ARGS_URL
+  value: https://test-domain.region.es.amazonaws.com
+```
+
+:::note
+Don't forget to set the `serviceAccountName` of the deployment/statefulset to the created service account with the IRSA annotation.
+:::
+
+## Troubleshooting
+
+### Versions used
+
+This page was created based on the following versions available and may work with newer releases of mentioned software.
+
+| Software                                                                                                                                              | Version      |
+| ----------------------------------------------------------------------------------------------------------------------------------------------------- | ------------ |
+| AWS Aurora PostgreSQL                                                                                                                                 | 13 / 14 / 15 |
+| AWS OpenSearch                                                                                                                                        | 2.5          |
+| [Aws-advanced-jdbc-wrapper](https://github.com/awslabs/aws-advanced-jdbc-wrapper)                                                                     | 2.2.2        |
```

Comment:
Let's use a normal name like the rest of the items in the list here.

So it will be: [Amazon Web Services (AWS) JDBC Driver](https://github.com/awslabs/aws-advanced-jdbc-wrapper)